lib/schema/facter.yaml

```diff
@@ -184,6 +184,15 @@ chassistype:
     caveats: |
         Linux: kernel 2.6+ is required due to the reliance on sysfs.
 
+cloud:
+    type: map
+    description: Information about the cloud instance of the node. This is currently only populated on linux nodes running in Microsoft Azure.
```

Comment:
@gguillotte as far as I can tell we usually capitalize Linux. Should that also be the case here?